src/player.js

```diff
@@ -0,0 +1,274 @@
+const pieceId = 'position_';
+
+export default class Player {
+    #_position = 0;
```

Comment:
Usage of `#` is not recommended in variable/property names. If it is an internal variable/property, just use the `_` prefix.